backend/src/app/migrations/sql/0067-add-team-invitation-table.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE team_invitation (  
+  team_id uuid NOT NULL REFERENCES team(id) ON DELETE CASCADE,
+  email_to text NOT NULL,
+  role varchar(255) NOT NULL,
```

Comment:
don't use varchar, should be text, also remember to mark text fields as external.

example from audit_log table:
```sql
ALTER TABLE audit_log
  ALTER COLUMN name SET STORAGE external,
  ALTER COLUMN type SET STORAGE external,
  ALTER COLUMN props SET STORAGE external;
  ```

Reply:
fixed